### DISCUSSION TITLE
error: Não há versões disponíveis do pacote 'Microsoft.EntityFrameworkCore.Sqlite'

### DISCUSSION THREAD

**Discussion by DiegoFernandes26** [balta-io/2811#21] [Q&A] [answered]
Ola! Eu mudei de computador, e estou tendo o seguinte erro ao tentar instalar o EntityFrameworkCore. 

PS C:\Users\diego\OneDrive\Área de Trabalho\Teste> dotnet add package Microsoft.EntityFrameworkCore.Sqlite         
  Determinando os projetos a serem restaurados...
  Writing C:\Users\diego\AppData\Local\Temp\tmpD583.tmp
info : Adicionando PackageReference do pacote 'Microsoft.EntityFrameworkCore.Sqlite' ao projeto 'C:\Users\diego\OneDrive\Área de Trabalho\Teste\Teste.csproj'.
error: Não há versões disponíveis do pacote 'Microsoft.EntityFrameworkCore.Sqlite'

Estou com a versao 6.0.3 do .Net instalado. Alguem poderia me dizer como resolver esse problema...nos outros computadores nunca tive esse problema.

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @DiegoDevFer, como vai?

Não sei por que, mas algumas vezes quando instalamos o .NET no Windows, ele não registra o Nuget.org como fonte de pacotes automaticamente.

Desta forma, você precisa executar esta linha aqui no seu terminal antes:
```
dotnet nuget add source https://api.nuget.org/v3/index.json -n nuget.org
```

Feito isto, basta executar um `dotnet restore` no seu projeto que deve funcionar.

**Reply by DiegoFernandes26** [reply]
Cara, muito bom! eu ja havia pesquisado de todo jeito mas nao encontrava respostas. Obrigado pela rapidez na resposta!!

**Reply by oseiasmribeiro** [reply]
@andrebaltieri Muito obrigado!

**Reply by SamirMMBr** [reply]
Show, estava com mesmo erro no pacote dotnet add package Azure.Storage.Blobs e resolveu.
